### ISSUE TITLE
Imprevement: add support to simulate start / stop charging easily

### ISSUE THREAD

**Issue by henrywah196** [nenecmrf/OCPP-J-CP-Simulator#7]
Let's say a single button click to start charging, which include send StatusNotification, StartTransaction, then  MeterValues with the interval. after a while click a single button to stop charging, which will send StopTransaction and StatusNotification.
With this simulation, we should simulate energy usage during charging. such as energy, power, current, voltage, etc.

**Comment by psnehanshu** [comment]
That would be nice? Btw, are you building something, @henrywah196 ?

**Comment by kubarskii** [comment]
@henrywah196 it s a good idea, but the main purpose of the sim is to test the ocpp server))
but you can always build such simulator using this one

**Comment by henrywah196** [comment]
Yes. we are building a central system. I'm a QA tester and using this tool for testing. thanks for the good work. Yes. I've rebuild it using this to meet my requirements.

**Comment by johan-bell** [comment]
> Yes. we are building a central system. I'm a QA tester and using this tool for testing. thanks for the good work. Yes. I've rebuild it using this to meet my requirements.

Is it possible to have your central system please ? In Which language is it ?
Will be so thankful to see it !

**Comment by psnehanshu** [comment]
@belljohan3 I have some boilerplate code to help you get started. It's in node.js https://github.com/pSnehanshu/ocpp-central-system-boilerplate-code

**Comment by johan-bell** [comment]
Thanks so much @pSnehanshu. I will have a look at that project. Please, just stay connected. so that if i get some issuses you can help me please
